src/odemis/driver/xt_client.py

```diff
@@ -1157,7 +1168,7 @@ def _moveTo(self, future, pos, timeout=60):
                 moving = True
                 tstart = time.time()
                 while moving:
-                    pos = self.parent.get_stage_position()
+                    pos = self._getPosition()
                     moving = self.parent.stage_is_moving()
                     # Take the opportunity to update .position
                     self._updatePosition(pos)
```

Comment:
If you pass the result of getPosition, which already has the axes inverted to updatePosition(), the axes are going to be inverted twice. => not inverted.